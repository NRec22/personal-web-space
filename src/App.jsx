import React from 'react';

const App = () => <h1>Hello, world!</h1>;

const Hello = () => 'Hello, world!';

export { App, Hello };
