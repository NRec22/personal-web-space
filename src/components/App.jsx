import React, { Fragment } from 'react';

const App = () => (
  <Fragment>
    <h1>Neven Recchia</h1>
    <h2>Home</h2>
    <h2>About Me</h2>
    <h2>Contact</h2>
    <h2>LinkedIn</h2>
    <h2>Github</h2>
  </Fragment>
);

export default App;
