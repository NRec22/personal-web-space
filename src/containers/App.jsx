import React, { Fragment } from 'react';
import Header from '../components/Header';
import Banner from '../components/Banner';

const App = () => (
  <Fragment>
    <Header />
    <Banner />
  </Fragment>
);

export default App;
