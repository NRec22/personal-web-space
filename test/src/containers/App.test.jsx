import { shallow } from 'enzyme';
import React from 'react';
import toJson from 'enzyme-to-json';
import App from '../../../src/containers/App';

describe('App:', () => {
    it('should contain my name', () => {
        const wrapper = shallow(<App />);
        const myName = 'Neven Recchia';

        expect(wrapper.text()).toContain(myName);
    });

    it('should have nav option text', () => {
        const wrapper = shallow(<App />);
        const navText = ['Home', 'About Me', 'Contact', 'LinkedIn', 'Github'];

        navText.forEach((text) => {
            expect(wrapper.text()).toContain(text);
        })
    });

    it('should render as expected', () => {
        const wrapper = shallow(<App />);
        const tree = toJson(wrapper);
        expect(tree).toMatchSnapshot();
    })
});
