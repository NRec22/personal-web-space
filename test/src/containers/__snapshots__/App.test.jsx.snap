// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App: should render as expected 1`] = `
<React.Fragment>
  <h1>
    Neven Recchia
  </h1>
  <h2>
    Home
  </h2>
  <h2>
    About Me
  </h2>
  <h2>
    Contact
  </h2>
  <h2>
    LinkedIn
  </h2>
  <h2>
    Github
  </h2>
  <Columns
    breakpoint={null}
    centered={false}
    className=""
    gapless={false}
    multiline={true}
    style={Object {}}
  >
    <Columns.Column
      className=""
      desktop={
        Object {
          "narrow": false,
          "offset": null,
          "size": null,
        }
      }
      desktopOffset={null}
      desktopSize={null}
      fullhd={
        Object {
          "narrow": false,
          "offset": null,
          "size": null,
        }
      }
      fullhdOffset={null}
      fullhdSize={null}
      mobile={
        Object {
          "narrow": false,
          "offset": null,
          "size": null,
        }
      }
      mobileOffset={null}
      mobileSize={null}
      narrow={false}
      offset={null}
      size={null}
      style={Object {}}
      tablet={
        Object {
          "narrow": false,
          "offset": null,
          "size": null,
        }
      }
      tabletOffset={null}
      tabletSize={null}
      widescreen={
        Object {
          "narrow": false,
          "offset": null,
          "size": null,
        }
      }
      widescreenOffset={null}
      widescreenSize={null}
    >
      Hello
    </Columns.Column>
    <Columns.Column
      className=""
      desktop={
        Object {
          "narrow": false,
          "offset": null,
          "size": null,
        }
      }
      desktopOffset={null}
      desktopSize={null}
      fullhd={
        Object {
          "narrow": false,
          "offset": null,
          "size": null,
        }
      }
      fullhdOffset={null}
      fullhdSize={null}
      mobile={
        Object {
          "narrow": false,
          "offset": null,
          "size": null,
        }
      }
      mobileOffset={null}
      mobileSize={null}
      narrow={false}
      offset={null}
      size={null}
      style={Object {}}
      tablet={
        Object {
          "narrow": false,
          "offset": null,
          "size": null,
        }
      }
      tabletOffset={null}
      tabletSize={null}
      widescreen={
        Object {
          "narrow": false,
          "offset": null,
          "size": null,
        }
      }
      widescreenOffset={null}
      widescreenSize={null}
    >
      Testing
    </Columns.Column>
    <Columns.Column
      className=""
      desktop={
        Object {
          "narrow": false,
          "offset": null,
          "size": null,
        }
      }
      desktopOffset={null}
      desktopSize={null}
      fullhd={
        Object {
          "narrow": false,
          "offset": null,
          "size": null,
        }
      }
      fullhdOffset={null}
      fullhdSize={null}
      mobile={
        Object {
          "narrow": false,
          "offset": null,
          "size": null,
        }
      }
      mobileOffset={null}
      mobileSize={null}
      narrow={false}
      offset={null}
      size={null}
      style={Object {}}
      tablet={
        Object {
          "narrow": false,
          "offset": null,
          "size": null,
        }
      }
      tabletOffset={null}
      tabletSize={null}
      widescreen={
        Object {
          "narrow": false,
          "offset": null,
          "size": null,
        }
      }
      widescreenOffset={null}
      widescreenSize={null}
    >
      Columns
    </Columns.Column>
  </Columns>
</React.Fragment>
`;
